contracts/PeeranhaCommunityToken.sol

```diff
@@ -0,0 +1,138 @@
+//SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "./interfaces/IPeeranhaCommunityToken.sol";
+import "./interfaces/IPeeranhaCommunityTokenFactory.sol";
+import "./libraries/CommonLib.sol";
+import "./base/NativeMetaTransaction.sol";
+
+import "@openzeppelin/contracts-upgradeable/token/ERC20/extensions/IERC20MetadataUpgradeable.sol";
+
+// transfer: 
+//    payable(userAddress).transfer(amount);
+//    IERC20Upgradeable(tokenAddress).transfer(userAddress, 2);
+//
+// balace:
+//    userAddress.balance
+//    IERC20Upgradeable(tokenAddress).balanceOf(userAddress)
+
+
+contract PeeranhaCommunityToken is IPeeranhaCommunityToken, NativeMetaTransaction {  
```

Comment:
Let's not use `Peeranha` in new contract names

Rename to `CommunityTokenRewards`